netbout-web/src/test/java/com/netbout/rest/TkInboxTest.java

```diff
@@ -53,29 +53,37 @@
 public final class TkInboxTest {
 
     /**
+     * Alias used in the tests.
+     */
+    private static final String ALIAS = "test";
+
+    /**
+     * URN of the identity used in the tests.
+     */
+    private static final String URN = "urn:test:1";
```

Comment:
@bdragan no need to share this variable among tests. let's have a different value for each test.